praw/models/reddit/mixins/replyable.py

```diff
@@ -11,6 +11,16 @@ def reply(self, body):
         :param body: The markdown formatted content for a comment.
         :returns: A :class:`~.Comment` object for the newly created comment.
 
+        Example usage:
+
+        .. code:: python
+
+            submission = reddit.submission(id='5or86n')
+            submission.reply(reply)
```

Comment:
Can we put `reply` in a string so non-astute readers aren't confused about why `reply` is not defined if they try to copy-paste this example?